risr-app/risr_proj/dashboard-fe/src/components/BarChart.js

```diff
@@ -62,9 +62,14 @@ class BarChart extends Component {
    */
   static getDerivedStateFromProps(nextProps, prevState) {
     // Check if data has been loaded.
-    if (!nextProps.data) return null;
+    // console.log(nextProps.data === {[]});
+    if (!nextProps.data || !nextProps.data.length) return null;
 
+    // Check if data has the expected keys.
     const {data} = nextProps;
+
+    if (!('pr_range' in data[0]) || !('repo_count' in data[0])) return null;
```

Comment:
add comment explaining this -- why are pr_range and repo_count required keys?

Reply:
At the moment, the bar chart component only works with one data set. Would it be alright if I just add a comment for now and then maybe change the component to work with more data sets later?

Reply:
sure!